<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="rasta" name="com.test1.tests.RTest" tests="24" failures="3" timestamp="5 Jan 2017 22:35:23 GMT" time="210.955" errors="0">
  <testcase name="verifyHomeSubmitEmailField" time="5.470" classname="com.test1.tests.RTest"/>
  <testcase name="isHomeEmailSubmissionSuccessful" time="6.565" classname="com.test1.tests.RTest"/>
  <testcase name="isShoeBlurbAvailable" time="20.665" classname="com.test1.tests.RTest">
    <failure type="java.lang.AssertionError" message="The shoe information blurb is not available. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The shoe information blurb is not available. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.test1.tests.RTest.isShoeBlurbAvailable(RTest.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- isShoeBlurbAvailable -->
  <testcase name="isEmailSubmissionSuccessful" time="7.351" classname="com.test1.tests.RTest"/>
  <testcase name="isShoePictureAvailable" time="19.799" classname="com.test1.tests.RTest">
    <failure type="java.lang.AssertionError" message="The shoe picture is not available. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The shoe picture is not available. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.test1.tests.RTest.isShoePictureAvailable(RTest.java:86)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- isShoePictureAvailable -->
  <testcase name="verifySubmitEmailField" time="6.321" classname="com.test1.tests.RTest"/>
  <testcase name="verifySubmitEmailField" time="6.118" classname="com.test1.tests.RTest"/>
  <testcase name="isShoePriceAvailable" time="19.839" classname="com.test1.tests.RTest">
    <failure type="java.lang.AssertionError" message="The shoe price is not available. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The shoe price is not available. expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.test1.tests.RTest.isShoePriceAvailable(RTest.java:75)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- isShoePriceAvailable -->
  <testcase name="isShoePictureAvailable" time="6.968" classname="com.test1.tests.RTest"/>
  <testcase name="verifySubmitEmailField" time="6.010" classname="com.test1.tests.RTest"/>
  <testcase name="isShoePriceAvailable" time="7.241" classname="com.test1.tests.RTest"/>
  <testcase name="isEmailSubmissionSuccessful" time="6.956" classname="com.test1.tests.RTest"/>
  <testcase name="isShoeBlurbAvailable" time="6.814" classname="com.test1.tests.RTest"/>
  <testcase name="isHomeEmailSubmissionSuccessful" time="6.713" classname="com.test1.tests.RTest"/>
  <testcase name="isHomeEmailSubmissionSuccessful" time="7.141" classname="com.test1.tests.RTest"/>
  <testcase name="isShoePriceAvailable" time="9.224" classname="com.test1.tests.RTest"/>
  <testcase name="isEmailSubmissionSuccessful" time="7.072" classname="com.test1.tests.RTest"/>
  <testcase name="verifySubmitEmailField" time="5.605" classname="com.test1.tests.RTest"/>
  <testcase name="isShoePictureAvailable" time="10.256" classname="com.test1.tests.RTest"/>
  <testcase name="isEmailSubmissionSuccessful" time="7.708" classname="com.test1.tests.RTest"/>
  <testcase name="isEmailSubmissionSuccessful" time="7.310" classname="com.test1.tests.RTest"/>
  <testcase name="isEmailSubmissionSuccessful" time="7.822" classname="com.test1.tests.RTest"/>
  <testcase name="isShoeBlurbAvailable" time="8.881" classname="com.test1.tests.RTest"/>
  <testcase name="isEmailSubmissionSuccessful" time="7.106" classname="com.test1.tests.RTest"/>
</testsuite> <!-- com.test1.tests.RTest -->
